blueprints/ember-cli-mocha/index.js

```diff
@@ -13,14 +13,17 @@ module.exports = {
     var addonContext = this;
 
     return this.addBowerPackagesToProject([
-      { name: 'mocha',                 source: 'mocha',                           target: '~2.2.4' },
-      { name: 'chai',                  source: 'chai',                            target: '~2.3.0' },
-      { name: 'ember-mocha-adapter',   source: 'ember-mocha-adapter',             target: '~0.3.1' },
-      { name: 'ember-mocha',           source: 'ember-mocha',                     target: '~0.8.8' },
-      { name: 'ember-cli-test-loader', source: 'ember-cli/ember-cli-test-loader', target: '0.2.2'  }
+      { name: 'mocha',                 source: 'mocha',                 target: '~2.2.4' },
+      { name: 'chai',                  source: 'chai',                  target: '~2.3.0' },
+      { name: 'ember-mocha-adapter',   source: 'ember-mocha-adapter',   target: '~0.3.1' },
+      { name: 'ember-mocha',           source: 'ember-mocha',           target: '~0.8.8' },
+      { name: 'ember-cli-test-loader', source: 'ember-cli-test-loader', target: '0.2.2'  }
     ]).then(function() {
       if ('removePackageFromProject' in addonContext) {
-        return addonContext.removePackageFromProject('ember-cli-qunit');
+        return addonContext.removePackageFromProject([
```

Comment:
there's actually an `s` missing. it should be `removePackagesFromProject()`. `removePackageFromProject()` will only work with a single package name.